app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+      </ol>
+    </p>
+  </section>
+
+  <hr>
+
+  <section class="contributing-areas">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Make Up Exercises</h2>
+
+          <p>
+            If you have an idea for an exercise you can go to the <a href="https://github.com/exercism/x-common">x-common</a> repository, there you will find existing exercises and can check if your exercise is not already defined.
+          </p>
+
+           <p>
+            You can check <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">here</a> to learn how to implement a new exercise.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Translate Exercises</h2>
+
+          <p>
+            Translating an exercise to your favorite language is a great way to help the comunity, track maintainers will be more than happy to give all the support needed so you can complete your first contribution.
+          </p>
+
+          <p>
+            A detailed guide can be seen <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">here</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Review Pull Requests</h2>
+
+          <p>
+            Reviewing Pull Requests is a great way to both learn and teach a programming language, the track maintainers will really appreciate your help!
+          </p>
+
+          <p>
+             You can search your language repository <a href="https://github.com/exercism/">here</a>.
+          </p>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Canonical Data</h2>
+
+          <p>What is this section about?</p>
+          <p>A link to further instructions.</p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Help Triage Issues</h2>
+
+          <p>What is this section about?</p>
+          <p>A link to further instructions.</p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Fix Bugs</h2>
+
+          <p>What is this section about?</p>
+          <p>A link to further instructions.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Add Features</h2>
+
+          <p>What is this section about?</p>
+          <p>A link to further instructions.</p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Design Stuff</h2>
```

Comment:
This is a great point, in no way we want people to think that improving the UX isn't important.

I like `Improve user experience` better than `Improve UX`, but it will not fit in a single line unfortunately. I don't know exactly why, but I don't like `Improve UX` so much, but if our current title passes the wrong impression I'm all about changing it. =]